index.html

```diff
@@ -63,7 +64,7 @@ <h3 class="multi_story">Multi-Post Stories</h3>
   </div>
 
   <div class="recent_work_container flex">
- <div class="first_project">
+    <div class="first_project">
     <h3 class="first_project_title">Profesional Art Printing Data</h3>
     <p class="first_project_content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
     <ul class="first_project_skill_list flex">
```

Comment:
- [ ] I would suggest you please stick to the design as much as possible. your logo and content should be inline on the same line and also increase the width of the `p` tag so it matches the design. See the image below for reference. The text is aligned to the left but it doesn't pass the middle of the page.
### Your design
![his](https://user-images.githubusercontent.com/81727988/206657205-0974fb90-fb57-40b0-a528-20b18dd611a1.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/206657242-10e1755b-abb6-4a3e-b281-4ef70ea1fc8c.png)